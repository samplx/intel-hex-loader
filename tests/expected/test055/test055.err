ERROR: Unable to load patch file tests/com-files/this-file-does-not-exist.com
