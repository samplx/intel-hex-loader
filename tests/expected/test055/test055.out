initial memoryLimit  = 0x0100
program size setting = none
loading COM file     < 'tests/com-files/this-file-does-not-exist.com'
